Glideslope.json

```diff
@@ -30297,10 +30284,10 @@
    "associatedRunwayNumber": "26",
    "frequency": 109300,
    "elevation": 978,
-   "latitude": 48.23390478,
-   "longitude": 14.20774842,
-   "bearing": 267.04,
-   "glideslope": 3.0026704,
+   "latitude": 48.23392055,
+   "longitude": 14.20774199,
+   "bearing": 267.02,
+   "glideslope": 3.26702,
```

Comment:
Check this one, looks like you've put the start of the glideslope then the heading with no rest of the glideslope.